module/chunks/chunkVerifier_test.go

```diff
@@ -262,15 +301,24 @@ func GetBaselineVerifiableChunk(t *testing.T, script string) *verification.Verif
 
 	// events
 	chunkEvents := make(flow.EventsList, 0)
-	for i := 0; i < collectionSize; i++ {
-		if i == magicTxIndex {
-			switch script {
-			case "failedTx":
-				continue
+
+	erServiceEvents := make([]flow.ServiceEvent, 0)
+
+	if system {
+		chunkEvents = flow.EventsList{}
+		erServiceEvents = serviceEventsList
+	} else {
+		for i := 0; i < collectionSize; i++ {
+			if i == magicTxIndex {
+				switch script {
+				case "failedTx":
```

Comment:
could add a constant for this

Reply:
Yeah, but whole test relies on those simple strings, I think that's fine to keep it consistent